Tests/SwerverTests/RequestSpec/HttpRequestSpec.swift

```diff
@@ -0,0 +1,22 @@
+import Quick
+import Nimble
+import Swerver
+import Foundation
```

Comment:
I would check all your uses of `import Foundation` and make sure they are needed.